src/triangle.h

```diff
@@ -8,37 +8,33 @@
 #include "interval.h"
 #include "aabb.h"
 #include "material.h"
+#include "HostScene.h"
 
-class triangle: public hitable
-{
+class triangle {
 public:
-    __device__ triangle(float3 v0, float3 v1, float3 v2, material *mat_ptr) : v0(v0), v1(v1), v2(v2), mat_ptr(mat_ptr) {
-        interval x_interval = interval(min(v0.x,min(v1.x,v2.x)),max(v0.x,max(v1.x,v2.x)));
-        interval y_interval = interval(min(v0.y,min(v1.y,v2.y)),max(v0.y,max(v1.y,v2.y)));
-        interval z_interval = interval(min(v0.z,min(v1.z,v2.z)),max(v0.x,max(v1.z,v2.z)));
-        bbox = aabb(x_interval, y_interval, z_interval);
```

Comment:
Hello, where is by bbox? really important for bvh (which i know doesnt work but it will (someday))

Reply:
It works now!!!!!!!!!!!!!!